drupal-ukraine/advanced_forum
# Advanced forum d8

Initial module, just info file

merge after #5 

Steps for review
- run sh reinstall.sh 
- check Advanced Forum module enabled

![image](https://cloud.githubusercontent.com/assets/563412/11881321/c317e5cc-a50e-11e5-9681-f3042a114035.png)
